canvas_sdk/protocols/clinical_quality_measure.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+from canvas_sdk.protocols.base import BaseProtocol
+
+
+class ClinicalQualityMeasure(BaseProtocol):
+    """
+    The class that ClinicalQualityMeasure protocols inherit from.
+    """
+
+    class Meta:
+        title: str = ""
+        identifiers: list[str] = []
+        description: str = ""
+        information: str = ""
+        references: list[str] = []
+        types: list[str] = []
+        authors: list[str] = []
+        show_in_chart: bool = True
+        show_in_population: bool = True
+        can_be_snoozed: bool = True
+        is_abstract: bool = False
+
```

Comment:
Why are all of these identifiers under `Meta`?

Reply:
these are the same identifiers used in the workflow kit that categorize the cqms. they map to fields on the ProtocolUpload and ProtocolUploadVersion tables.